Makefile

```diff
@@ -12,7 +12,7 @@ REPO_PATH := github.com/deis/${SHORT_NAME}
 
 # The following variables describe the containerized development environment
 # and other build options
-DEV_ENV_IMAGE := quay.io/deis/go-dev:v0.22.0
+DEV_ENV_IMAGE := quay.io/deis/go-dev:v1.10.0
```

Comment:
Very nice update of the dev_env image. :+1:

Reply:
Yeah, this is a conservative change. We could probably use the newest version, but it uses `ubuntu:xenial` instead `deis/base:v0.3.6` (also a stripped down xenial).

Reply:
Let's start conservatively for now and then we can move up. :+1: